.github/workflows/pull_request.yml

```diff
@@ -39,6 +39,12 @@ jobs:
     - run: bundle exec rbs collection install
     - run: bundle exec rbs -I sig validate
 
+    - name: build and show content
+      run: |
+       bundle exec rake build
+       tar -xvf pkg/line-bot-api-*.gem
+       tar -zxvf data.tar.gz
```

Comment:
`bundle exec rate build`  may fail in release, so let me add this step to detect failure in PR.